addons/sale_loyalty/models/loyalty_reward.py

```diff
@@ -13,4 +13,5 @@ def _get_discount_product_values(self):
                 'taxes_id': False,
                 'supplier_taxes_id': False,
             })
+            vals['invoice_policy'] = 'order'
```

Comment:
Could you directly set it in the update just above? =]

Reply:
Hello @Demesmaeker ! I implemented your suggestion, added tests to check the invoice_policy, and deleted the manual assignment of the invoice_policy in the tests